cluster/openstack-heat/kubernetes-heat/fragments/configure-salt.yaml

```diff
@@ -32,6 +32,10 @@ write_files:
       password=$OS_PASSWORD
       region=$OS_REGION_NAME
       tenant-id=$OS_TENANT_ID
+      [LoadBalancer]
+      lb-version=$LBAAS_VERSION
+      subnet-id=$SUBNET_ID
```

Comment:
So for this change we have 2 choices:

- Default to v1, as written.
- Default to v2 above at line 38.

Defaulting to v2 here would effectively be a change in the heat scripts vs the regular k8s openstack plugin that defaults to v1 if nothing is specified.  I agree 100% that at some point we need to start defaulting to v2, and that point is now / very soon.  I kind of think the two pieces of k8s/openstack should follow the same logic, whatever that logic is, however - so I have a weak preference for that s/v1/v2/g default to be in another change.

I'm working on a PR to just query the available neutron extensions, and pick v2 vs v1 automatically - so we can hopefully drop all the explicit defaults once that is merged and never speak of this again.  I don't have an opinion on whether we should delay this change until that other one lands - I'm fine with merging this change and then removing the explicit default (line 38) later if there's any urgency.